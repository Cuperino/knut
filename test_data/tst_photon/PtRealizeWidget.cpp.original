void bah() {
    PtWidget_t *containerOS_widgetMpeg;
    PtRealizeWidget  (  containerOS_widgetMpeg);
}
