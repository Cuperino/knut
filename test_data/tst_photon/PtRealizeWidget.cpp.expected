void bah() {
    Photon::Widget *containerOS_widgetMpeg;
    containerOS_widgetMpeg->show();
}
